utils/utils-lib/src/constants.rs

```diff
@@ -83,3 +83,14 @@ pub const DEFAULT_REST_MAX_WORKER_THREADS: &str = "8";
 
 /// The default kubernetes namespace for this project.
 pub const DEFAULT_NAMESPACE: &str = "mayastor";
+/// NQN prefix for NVMe targets created by the product.
+pub const NVME_TARGET_NQN_PREFIX: &str = "nqn.2019-05.io.openebs:";
+
+/// NVMe path check period.
+pub const NVME_PATH_CHECK_PERIOD: &str = "3s";
+
+/// The default retransmission interval for reporting failed paths in case of network issues.
+pub const NVME_PATH_RETRANSMISSION_PERIOD: &str = "3s";
+
+/// Period for aggregating multiple failed paths before reporting them.
+pub const NVME_PATH_AGGREGATION_PERIOD: &str = "1s";
```

Comment:
3 seconds seems quite often, was this intentional?

Reply:
What do you suggest ? 5 secs ? 10 secs ?

Reply:
I don't know :D My punt would be 10s, but I don't really know what the best thing to do here is, @GlennBullingham any suggestion?

Reply:
I will change it to 10 seconds for now, we can adjust the value later, if needed.

Reply:
And it's a cmdline argument anyway so we can adjust on the fly if needed which is nice.

Reply:
Ah that's interesting, if this agent restarts, can the path ever disappear completely from the node while the agent is restarting? And in such case, the agent will not report the path as broken?